.github/workflows/labels.yml

```diff
@@ -101,10 +134,14 @@ jobs:
               },
               async (response, done) => (await Promise.allSettled(response.data.map(async (pull_request) => {
                 try {
-                  const log = (k,v) => core.info(`PR #${pull_request.number} - ${k}: ${v}`)
+                  const log = (k,v,skip) => {
+                    core.info(`PR #${pull_request.number} - ${k}: ${v}` + skip ? ' (skipped)' : '')
+                    return skip
+                  }
```

Comment:
Interesting that all uses of `log` make sense to accept a `skip` parameter... Feels like a strange abstraction, but it works here :+1: